mail-manager/pom.xml

```diff
@@ -2,19 +2,22 @@
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>inc.softserve</groupId>
-  <artifactId>EmailService</artifactId>
-  <version>1.0-SNAPSHOT</version>
+  <parent>
+    <groupId>inc.softserve</groupId>
+    <artifactId>email-service</artifactId>
+    <version>0.1-alpha</version>
+  </parent>
+
+  <artifactId>mail-manager</artifactId>
   <packaging>jar</packaging>
 
-  <name>EmailService</name>
-  <url>http://maven.apache.org</url>
+  <name>Email requests processor</name>
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <dropwizard.version>0.8.0-rc1</dropwizard.version>
     <guice.version>3.0</guice.version>
-    <email-core.version>1.0-SNAPSHOT</email-core.version>
+    <email-model.version>0.1-alpha</email-model.version>

```

Comment:
Do you mind against using `project.version` instead of keeping separate version for dependencies?